components/navbar/index.tsx

```diff
@@ -5,18 +5,35 @@ export interface NavBarProps {}
 
 const NavBar: FC<NavBarProps> = () => {
   return (
-    <div className="NavBar">
-      <NavItems />
+    <div>
+      <header className="NavBar">
+        <nav>
+          <ul className="NavBar__UL">
+            <NavItems />
+          </ul>
+        </nav>
+      </header>
       <style jsx>
         {`
+          .header {
+            margin: 0 auto;
+            padding: 1rem;
```

Comment:
What is this class referencing? You gave the header tag a className, and you have that css defined. Why also attempt to target its styling with this?